src/lib.rs

```diff
@@ -10,11 +10,23 @@
 
 #[macro_use]
 extern crate bitflags;
-extern crate core_foundation_sys;
-pub extern crate coreaudio_sys as sys;
 
 pub use error::Error;
 
 #[cfg(feature = "audio_unit")]
 pub mod audio_unit;
 pub mod error;
+
+pub mod sys {
+    #[cfg(feature = "audio_toolbox")]
+    pub use objc2_audio_toolbox::*;
+    #[cfg(feature = "core_audio")]
+    pub use objc2_core_audio::*;
+    #[cfg(feature = "core_audio")]
+    pub use objc2_core_audio_types::*;
+    #[cfg(feature = "core_midi")]
+    pub use objc2_core_midi::*;
+
+    // MacTypes.h
+    pub type OSStatus = i32;
+}
```

Comment:
I'm not a big fan of this module, ideally users should probably import the crates themselves, but it's useful to not break too much at once.